src/macros.rs

```diff
@@ -27,17 +27,17 @@
 ///     data.0, data.1, private_data);
 /// # }
 /// ```
-#[macro_export(local_inner_macros)]
+#[macro_export]
 macro_rules! log {
     // log!(target: "my_target", Level::Info, key1 = 42, key2 = true; "a {} event", "log");
     (target: $target:expr, $lvl:expr, $($key:tt = $value:expr),+; $($arg:tt)+) => ({
         let lvl = $lvl;
         if lvl <= $crate::STATIC_MAX_LEVEL && lvl <= $crate::max_level() {
             $crate::__private_api_log(
-                __log_format_args!($($arg)+),
+                ::std::format_args!($($arg)+),
```

Comment:
I think this should be `core`.
```suggestion
                ::core::format_args!($($arg)+),
```

Reply:
I don’t think this will change what `::std` means in macros.

Reply:
I have replaced `::std` with `$crate::__private_api::` since the only thing I can trust is `$crate`.